src/Stepper/Step.tsx

```diff
@@ -0,0 +1,111 @@
+import * as classnames from "classnames";
+import * as React from "react";
+import * as PropTypes from "prop-types";
+
+import "./Step.less";
+import CheckMark from "./CheckMark";
+import {FlexBox} from "../flex";
+import Exclamation from "./Exclamation";
+
+const propTypes = {
+  className: PropTypes.string,
+  title: PropTypes.string,
+  description: PropTypes.oneOfType([PropTypes.node]),
+  showTitle: PropTypes.bool,
+  showMessage: PropTypes.bool,
+  showDescription: PropTypes.bool,
+  id: PropTypes.number,
+  seekable: PropTypes.bool,
```

Comment:
up to you, based on preference, but this prop is essentially redundant, since we can rely on the presence of an onClick to decide whether or not we render a working button.

we don't want to render a button with no onClick handler anyway, so better to just use an `onClick` check as the deciding factor